requirements.txt

```diff
@@ -4,7 +4,8 @@ cffi==1.15.1
 charset-normalizer==3.1.0
 craft-cli==1.2.0
 craft-parts==1.20.0
-craft-providers==1.12.0
+#craft-providers==1.12.0
+git+https://github.com/syu-w/craft-providers.git@support-23.04#egg=craft-providers
```

Comment:
I'm guessing we're on merge hold for this until we get craft-providers 1.13 for https://github.com/canonical/craft-providers/pull/330

Reply:
We don't need that PR for 23.04, but need craft-providers 1.13.